test/fixtures/import-all-css/expected.js

```diff
@@ -1,8 +1,10 @@
+"use strict";
+
 require("element-ui/lib/style.css");
 
-var _Element = _interopRequireDefault(require("element-ui/lib")).default;
+var _lib = _interopRequireDefault(require("element-ui/lib"));
 
-function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
 
-console.log(_Element.Button);
-console.log(_Element);
+console.log(_lib["default"].Button);
+console.log(_lib["default"]);
```

Comment:
This is currently rendered as `console.log(_Element);`. (while _Element is not defined)